packages/react/src/time-input/TimeInput.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { StoryObj, Meta } from '@storybook/react';
+import { useState } from 'react';
+import { TimeInput, TimeInputProps, TimeValue } from './TimeInput';
+import { isValidTime } from './utils';
+
+const meta: Meta<typeof TimeInput> = {
+	title: 'Forms/TimeInput',
+	component: TimeInput,
+	args: {
+		label: 'Time',
+	},
+};
+
+export default meta;
+
+type Story = StoryObj<typeof TimeInput>;
+
+export const Basic: Story = {
+	render: Render,
+};
+
+export const Invalid: Story = {
+	args: {
+		value: { raw: '9:66' },
+	},
+	render: Render,
+};
+
+export const DifferentTimeFormat: Story = {
+	args: {
+		timeFormat: 'HH:mm',
+	},
+	render: Render,
+};
+
+function Render(props: TimeInputProps) {
+	const [value, setValue] = useState<TimeValue>(props.value || { raw: '' });
+
+	const onChange = (e: TimeValue) => {
```

Comment:
What do you think about changing `e` to `timeValue`? To me, `e` stands for `event` but we're not actually passing the raw `event` object to the `onChange` callback, but instead a convenience time value.

Reply:
Sounds good